filters.txt

```diff
@@ -287,8 +287,8 @@ _clickthru.swf?
 ||romea.cz/296x186.swf
 ||romea.cz/images/servis/banner
 ||ronnie.cz/cache/bannery/
-||rubiconproject.com$third-party,domain=aktualne.cz
-||imedia.cz/$script
+||rubiconproject.com$third-party
+||s.imedia.cz/js/szn-script.js
```

Comment:
Is there any reason for this revert?

Reply:
I made wrong merge. Already corrected it.

Reply:
This is a last commit, previous rule (`||imedia.cz/$script`) is the correct one.

Reply:
Corrected. Thanks for review